tools/harness-thci/OpenThread.py

```diff
@@ -1783,13 +1771,12 @@ def configBorderRouter(
             if P_Prefix is None:
                 P_Prefix = 0xfd007d037d037d03
 
-            P_Prefix = '%016x' % P_Prefix
```

Comment:
Hi Jane 

Removing this line will break some 1.2 test cases in my opinion. I was looking also at this 'configBorderRouter' method as well today and according to my observations, the issue is that GRL test scripts don't use consistent syntax for P_Prefix paramter passed to this method. Sometimes there is  '0x' added to this argument and sometimes not. It works for T1.2 tests due to 'if TESTHARNESS_VERSION == TESTHARNESS_1_2' condition and the fact, that for those tests P_Prefix is always preceded by '0x'. It is mentioned in the comment:

# TestHarness 1.1 converts 2001000000000000 to "2001000000000000" (it's wrong, but not fixed yet.)

Reply:
Hi @tomaszkob89 ,

Thank you for adding your observations.  I did see that comment but I had read that as referring to the 1.1 Test Harness.  This is not that harness.  This issue is occurring in the 1.2 test harness when running the 1.1 test cases.  This THCI should function for both 1.1 and 1.2 test harness test cases.  Is this something that can be resolved with conditionalizing this in the THCI to perform as above in for the 1.2 test harness running these 1.1 cases but performing in the existing way for 1.2 cases?  Is there a Jira ticket in threadgroup.atlassian.net tracking this issue?

Reply:
This should be an Harness issue. It seems much easier to fix in Harness instead.

Reply:
I can inquire about a verification of the consistency of formatting these.

Reply:
> I can inquire about a verification of the consistency of formatting these.

Thanks!

Reply:
I've filed DEV-2164.